<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowPersonnelWindow</class>
 <widget class="QDialog" name="ShowPersonnelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>407</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Avenir Next</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listaPersonnel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>511</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioWszyscy">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Wszyscy</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioSprzedawca">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sprzedawcy</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioMechanik">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Mechanicy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
